src/panels/lovelace/cards/hui-glance-card.ts

```diff
@@ -277,6 +277,9 @@ export class HuiGlanceCard extends LitElement implements LovelaceCard {
       </div>`;
     }
 
+    const name =
+      "name" in entityConf ? entityConf.name : computeStateName(stateObj);
```

Comment:
We should probably update all of them to:
```suggestion
    const name = entityConf.name ?? computeStateName(stateObj);
```